src/Microsoft.OData.Core/UriParser/Resolver/NormalizedModelElementsCache.cs

```diff
@@ -20,6 +20,11 @@ namespace Microsoft.OData.Edm
     /// </summary>
     internal sealed class NormalizedModelElementsCache
     {
+        /// <summary>
+        /// Provides a shared, normalized case-insensitive cache of model elements for the EDM core model.
+        /// </summary>
+        public static NormalizedModelElementsCache EdmCoreModelCacheInstance = new NormalizedModelElementsCache(EdmCoreModel.Instance);

```

Comment:
Make it readonly.

```suggestion
        public static readonly NormalizedModelElementsCache EdmCoreModelCacheInstance = new NormalizedModelElementsCache(EdmCoreModel.Instance);

```

nit: since this is already in a class called cache, we could simplify the model. But feel free to ignore this part

```suggestion
        public static readonly NormalizedModelElementsCache EdmCoreModelInstance = new NormalizedModelElementsCache(EdmCoreModel.Instance);
```

or 

```suggestion
        public static NormalizedModelElementsCache EdmCoreModelCache = new NormalizedModelElementsCache(EdmCoreModel.Instance);

```